### PR TITLE
documentation: add visualize & PyvisVisualizer documentation

### PR DESCRIPTION
**Pull request by ytlee93**
*Issue #, if available:*

*Description of changes:* documentation: add visualize & PyvisVisualizer documentation

*Testing done:*
tox -e black-format & tox -e flake8,pylint,twine,black-check,docstyle --parallel all
tox -r -e py39 tests/integ/sagemaker/lineage/test_lineage_visualize.py
tox -e py39 tests/unit/sagemaker/lineage

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/lineage/query.py [hunk 1]
```diff
@@ -230,7 +230,31 @@ class PyvisVisualizer(object):
...
+                            "Context": {
+                                "name": "Context",
+                                "style": {"background-color": "#ff9900"},
+                                "isShape": "False",
+                            },
+                            "StartArn": {
+                                "name": "StartArn",
+                                "style": {"shape": "star"},
+                                "isShape": "True",
+                                "symbol": "★",  # shape symbol for legend
```
Comment (jkasiraj) [inline src/sagemaker/lineage/query.py:254]: Are there further requirements for the shape symbol? Can it be any character, could it be multiple characters?

**Review by jkasiraj — APPROVED** [review]

File: src/sagemaker/lineage/query.py (hunk 1 above)
Comment (ytlee93) [reply src/sagemaker/lineage/query.py:254]: The valid shapes are specified here: https://pyvis.readthedocs.io/en/latest/documentation.html
`shape ([str](https://docs.python.org/3/library/stdtypes.html#str) (optional)) – The shape defines what the node looks like. There are two types of nodes. One type has the label inside of it and the other type has the label underneath it. The types with the label inside of it are: ellipse, circle, database, box, text. The ones with the label outside of it are: image, circularImage, diamond, dot, star, triangle, triangleDown, square and icon.`

File: src/sagemaker/lineage/query.py [hunk 2]
```diff
@@ -230,7 +230,31 @@ class PyvisVisualizer(object):
     """Create object used for visualizing graph using Pyvis library."""
```
Comment (Snoyark) [inline src/sagemaker/lineage/query.py:230]: Is it made clear anywhere that the user needs to install Pyvis (like https://w.amazon.com/index.php/Coral/Model/XML/Traits#Pagination says for Sklearn)?

Comment (ytlee93) [reply src/sagemaker/lineage/query.py:230]: Requirement added to visualize() documentation

**Review by Snoyark — APPROVED** [review]

**Pull request merged**